src/StatusQ/Components/StatusChatListItem.qml

```diff
@@ -16,9 +16,8 @@ Rectangle {
     property string chatId: ""
     property string categoryId: ""
     property string name: ""
-    property alias badge: statusBadge
```

Comment:
Please do not remove the `badge` API, this is needed in other places where we need more control over the component itself

Reply:
@PascalPrecht ok, no problem.